README.md

```diff
@@ -44,62 +141,57 @@ On Ubuntu, an install script is available on each release [here](https://github.
 ./semgrep-v0.9.0-ubuntu-generic.sh
 ```
 
-To try `semgrep` without installation, you can also run it via [Docker](https://docs.docker.com/install/):
+To try Semgrep without installation, you can also run it via [Docker](https://docs.docker.com/install/):
 
 ```
 docker run --rm -v "${PWD}:/home/repo" returntocorp/semgrep --help
 ```
 
-## Usage
+## Terminology
 
-### Example Usage
+A **rule** (or *check*) is a Semgrep pattern that can be used to tell match
+source code.
 
-Here is a simple Python example, `test.py`. We want to retrieve an object by ID:
+A **rule pack** (*check pack*) is a collection of Semgrep rules bundled
+together. For example, "Here are a bunch of Java checks I want to run."
 
-```python3
-def get_node(node_id, nodes):
-    for node in nodes:
-        if node.id == node.id:  # Oops, supposed to be 'node_id'
-            return node
-    return None
-```
+## Usage
```

Comment:
These may be more "Workflow" related. Thoughts on adapting this section be about workflows and use cases, and added a "Usage" section that is more focused on the CLI? See Bento:

* https://github.com/returntocorp/bento#usage
* https://github.com/returntocorp/bento#workflows

Reply:
Given the length of the README I could see some of these workflow items being sub-pages and getting some breathing space.